core/src/main/java/io/debezium/connector/cassandra/FileOffsetWriter.java

```diff
@@ -71,58 +84,84 @@ public FileOffsetWriter(String offsetDir) throws IOException {
 
         loadOffset(this.snapshotOffsetFile, snapshotProps);
         loadOffset(this.commitLogOffsetFile, commitLogProps);
+        this.executorService = Executors.newFixedThreadPool(1);
```

Comment:
Please use `io.debezium.util.Threads`

Reply:
for using this we need SourceConnector class. I use AbstractSourceConnector for begining and if it's not good tell me to some how use CasssandraXConnector class.